Gemfile

```diff
@@ -42,8 +40,7 @@ gem 'ace-rails-ap'
 # Serializers
 gem 'rdf-n3'
 gem 'jbuilder'
-gem 'yajl-ruby'
-gem 'google_hash'
```

Comment:
Where it all started. Doesn't support ruby 2.7.x, and shows only marginal improvements in hash speed, which are negligable in terms of real world performance.